### PR TITLE
Dact 745 director address

### PR DESCRIPTION
**Pull request by jbestwickch**
https://companieshouse.atlassian.net/browse/DACT-745
Residential address entry. See Jira for a list of optional and mandatory fields.

### CODE REVIEW

**Review by mdesaikns — APPROVED** [review]

File: src/main/java/uk/gov/companieshouse/officerfiling/api/controller/ValidationStatusControllerImpl.java [hunk 1]
```diff
@@ -44,6 +45,10 @@ public class ValidationStatusControllerImpl implements ValidationStatusControlle
     private boolean isTm01Enabled;
     @Value("${FEATURE_FLAG_ENABLE_AP01:true}")
     private boolean isAp01Enabled;
+    @Value("#{'${COUNTRY_LIST}'.split(';')}")
+    private List<String> countryList;
+    @Value("#{'${UK_COUNTRY_LIST}'.split(';')}")
```
Comment (PhilipGlover) [inline src/main/java/uk/gov/companieshouse/officerfiling/api/controller/ValidationStatusControllerImpl.java:50]: in https://github.com/companieshouse/chs-configs/pull/2462 I see `COUNTRY_LIST` but not `UK_COUNTRY_LIST`

Comment (rwalrondch) [reply src/main/java/uk/gov/companieshouse/officerfiling/api/controller/ValidationStatusControllerImpl.java:50]: I have added UK_COUNTRY_LIST to all env files.

Comment (PhilipGlover) [reply src/main/java/uk/gov/companieshouse/officerfiling/api/controller/ValidationStatusControllerImpl.java:50]: I see it now, but it's missing the `export` command https://github.com/companieshouse/chs-configs/blob/develop/cidev/officer-filing-api/env#L14

Comment (rwalrondch) [reply src/main/java/uk/gov/companieshouse/officerfiling/api/controller/ValidationStatusControllerImpl.java:50]: Now I know why my mum used to say Less Haste More Speed.....  export added. Cheers.

**Review by PhilipGlover — APPROVED** [review]
I haven't run the code, but the logic in the OfficerAppointmentValidator looks good to me.

**Pull request merged**